client-spark/spark2/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -188,6 +188,8 @@ public RssShuffleManager(SparkConf sparkConf, boolean isDriver) {
     // External shuffle service is not supported when using remote shuffle service
     sparkConf.set("spark.shuffle.service.enabled", "false");
     LOG.info("Disable external shuffle service in RssShuffleManager.");
+    sparkConf.set("spark.shuffle.reduceLocality.enabled", "false");
```

Comment:
Because our data is in the RSS cluster or HDFS, locatity is useless for us. If rack aware need this feature, we can turn on this feature when we use rack aware.

Reply:
+1. The locality awareness could be supported in the future.